tutorials/matching_stack_run_1.2i.ipynb

```diff
@@ -0,0 +1,514 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_afw_table(bands=['r',]):\n",
+    "    \"\"\"Initializes an afw table with magnitude column(s)\"\"\"\n",
+    "    def define_mag_columns(bands):\n",
+    "        \"\"\"Defines columns for given list of bands\"\"\"\n",
+    "        Coldef = namedtuple('Coldef', 'name type doc'.split())\n",
+    "        return [Coldef('mag_{}'.format(bp), float, '{}-magnitude'.format(bp)) for bp in bands]\n",
+    "    # Some minimal schema\n",
+    "    schema = afw_table.SourceTable.makeMinimalSchema()\n",
+    "    # Define magnitude columns\n",
+    "    mag_cols = define_mag_columns(bands=bands)\n",
+    "    # Add magnitude columns to schema\n",
+    "    for mag_coldef in mag_cols:\n",
+    "        schema.addField(mag_coldef.name, type=mag_coldef.type, doc=mag_coldef.doc)\n",
+    "    return afw_table.SourceCatalog(schema)\n",
+    "\n",
+    "truth_afw = make_afw_table(bands=['r',])\n",
+    "for id_, ra_, dec_, mag_ in zip(*[truth_dict[col] for col in truth_cols]):\n",
+    "    row = truth_afw.addNew()\n",
+    "    row.set('id', id_)\n",
+    "    row.set('coord_ra', afw_geom.Angle(ra_, afw_geom.degrees))\n",
+    "    row.set('coord_dec', afw_geom.Angle(dec_, afw_geom.degrees))\n",
+    "    row.set('mag_{}'.format(our_bp), mag_)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Matching Experiments\n",
+    "\n",
+    "Now that we have the tools we need, let's read in the Run 1.2i DRP catalog data.\n",
+    "\n",
+    "We first instantiate our Butler object. Butler manages a collection of datasets known as a \"repository.\" Each dataset has a `datasetType` identified by strings such as `'src'` or `'calexp'`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create data butlers for the src and coadd repositories.\n",
+    "repo_path = '/global/cscratch1/sd/desc/DC2/data/Run1.2i_globus_in2p3_20181217/w_2018_39/rerun/multiband'\n",
+    "butler = dp.Butler(repo_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Sources or Objects?\n",
+    "\n",
+    "We can consider visit-level src catalog data, in which case we would define `datasetType='src'` and provide a query, or `dataId`, to the butler with (`visit`, `raftName`, `sensorName`) ids; or we can consider coadd object data, for which we would define `datasetType='coadd'` and provide a `dataId` with (`filter`, `tract`, `patch`) ids.\n",
+    "\n",
+    "Somewhat different flux models are available in the Run1.2i data for src catalogs versus coadd catalogs. We use `'base_PsfFlux'` for the flux model, as they are available in both.\n",
```

Comment:
Use `slot_PsfFlux` instead of `base_PsfFlux`.
`slot_` is the data column required by the processing
`base_` is how it was computed.
For most cases `slot_` points to `base_` and this doesn't matter.  But conceptually `slot_` is what we want.